### ISSUE TITLE
Question about case compatibility?

### ISSUE THREAD

**Issue by jittapont** [GEIGEIGEIST/TOTEM#16]
I want to ask about the case compatibility between the USB version and BLE version. Can I use the USB version (https://github.com/GEIGEIGEIST/TOTEM/tree/main/case/USB) first and change to wireless option with the same case?
Thanks a lot

**Comment by spamwax** [comment]
I believe it is compatible. Just make sure you disconnect the TRS cable between the two halves when you install the battery.

**Comment by isopix** [comment]
What happens if we forgot to disconnect it? Can some components be damaged?

**Comment by spamwax** [comment]
> What happens if we forgot to disconnect it? Can some components be damaged?

Yes, according to the build [guide](https://github.com/GEIGEIGEIST/TOTEM/blob/main/docs/buildguide.md#trrs-jacks-optional), it can damage it.
